src/bioregistry/data/bioregistry.json

```diff
@@ -1700,6 +1700,33 @@
     },
     "name": "Assembling the Fungal Tree of Life - Taxonomy"
   },
+  "agd": {
+    "contact": {
+      "email": "18894004819@126.com",
```

Comment:
this looks like a fake email address, a better way might be to look at the publication and find the corresponding author's email. Try doing control-f on https://www.frontiersin.org/journals/plant-science/articles/10.3389/fpls.2024.1445365/full for "correspondence" to see some options

Reply:
Updated the name and email to one of the authors of the paper. There are 2 other authors listed with valid email addresses that could be added as additional contacts if needed.